include/aws/auth/private/sso_token_providers.h

```diff
@@ -0,0 +1,97 @@
+#ifndef AWS_AUTH_TOKEN_PROVIDERS_PRIVATE_H
+#define AWS_AUTH_TOKEN_PROVIDERS_PRIVATE_H
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/auth/auth.h>
+#include <aws/auth/credentials.h>
+
+/**
+ * Configuration options for a provider that sources sso token information from the aws profile (by default
+ * ~/.aws/config) and token from ~/.aws/sso/cache/<sha1 of start url>.json.
```

Comment:
Worth to document this is the legacy way?

Reply:
Documented as public documentation also mentions that this is the legacy way.